index.html

```diff
@@ -16,7 +16,9 @@
             <img src="./logos/menu.svg" alt="navigation icon" class="navigation__hamburger">
```

Comment:
- [ ] You are using ```<h1>``` more than once, please use ```<h1>``` just one time for in your page, and use ```<h2>``` to ```<h6>``` for other heading in a meaningful hierarchy so your coded respect the accessibility rules.